### DISCUSSION TITLE
[Request] Max seed ratio and remove

### DISCUSSION THREAD

**Discussion by SecT0uch** [jesec/flood#303] [Ideas]
That would be so practical, especially in Feeds > Download rules.

When the torrent is added and downloaded, it would be practical if flood could stop seeding (and eventually remove) the torrent after reaching a chosen ratio.

**Comment by jesec** [comment]
It is on the roadmap. 

The biggest technical challenge is that Flood would have to poll the torrent list constantly, which could be resource-consuming. I am thinking about an event-based approach.
